### ISSUE TITLE
bug: Youtube goes black when minimized.

### ISSUE THREAD

**Issue by eria-tarka** [ReVanced/revanced-patches#2991]
### Type

Error at runtime

### Bug description

First of all, please apologize. My English is not very good. my main language is Spanish

so...

When a video plays, and the home key is pressed and the floating window closes, when you open the window again the video goes black.

### Steps to reproduce

open youtube
play any video
Press the "home" key on your cell phone to continue playing in the background
close the minimized window
open youtube again
The video is black although the audio continues

At this point if you minimize playback again, the video freezes.

### Relevant log output

```shell
None
```

### Screenshots or videos

https://github.com/ReVanced/revanced-patches/assets/144094120/8c9f0cd4-fc2a-4896-99fe-c64032ed8a00

### Additional context

I am using revanced manager 1.9.5 to patch, and youtube v18.32.39 with the default patches.
In version v18.23.35 this bug does not happen.
I use a motorola with android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Try on stock YouTube, same version.

**Comment by eria-tarka** [comment]
Unfortunately I don't have a premium subscription to try it in the stock version... but I tried it on another Samsung phone with Android 11 and the same thing happens. I also tried with revanced X and the same thing happens. I also tried version 18.29.38 with the same patches and same version of manager and the bug does not occur. It seems that the bug only occurs from version 18.32.39 and later. I'm waiting for a next update to see if the bug still occurs. Maybe it's a bug in the Android version, or in YouTube itself?

**Comment by eria-tarka** [comment]
> Try on stock YouTube, same version.

Unfortunately I don't have a premium subscription to try it in the stock version... but I tried it on another Samsung phone with Android 11 and the same thing happens. I also tried with revanced X and the same thing happens. I also tried version 18.29.38 with the same patches and same version of manager and the bug does not occur. It seems that the bug only occurs from version 18.32.39 and later. I'm waiting for a next update to see if the bug still occurs. Maybe it's a bug in the Android version, or in YouTube itself?
